addons/website/static/tests/tours/page_manager.js

```diff
@@ -109,3 +109,40 @@ wTourUtils.registerWebsitePreviewTour('website_page_manager', {
         run: () => null,
     },
 ]);
+
+wTourUtils.registerWebsitePreviewTour('website_page_manager_session_forced', {
+    test: true,
+    url: '/',
+}, [{
+    content: "Open the website switcher",
+    trigger: '.o_website_switcher_container button',
+}, {
+    content: "Switch to website 2",
+    // Ensure data-website-id exists
+    extra_trigger: 'iframe html[data-website-id="1"]',
+    trigger: '.o_website_switcher_container .dropdown-item:contains("My Website 2")'
+}, {
+    content: "Wait for the iframe to be loaded",
+    // The page reload generates assets for website 2, it may take some time
+    timeout: 20000,
+    trigger: 'iframe html:not([data-website-id="1"])',
```

Comment:
Can't you rather check that the value is actually 2 ?

Reply:
Yep, seems a bit weird, but it's just a copy paste of website switcher steps from another test.
I changed it a bit (but still used the `:not()`)